flowmachine/flowmachine/features/subscriber/distance_counterparts.py

```diff
@@ -120,6 +121,10 @@ def __init__(
 
         super().__init__()
 
+    @property
+    def column_names(self) -> List[str]:
+        return ["subscriber", f"distance_{self.statistic}"]
```

Comment:
Another note to self: we should work towards removing these types of column names that depend on the parameter name. It would simplify a lot of code if the second column here (and in other classes) was simply named `value`. It would also make the life of the caller much easier because they don't need a case distinction in their code to consume the resulting table. (After all, they should _know_ which statistic they requested.)